<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QWidget" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="mainMenuBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>840</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameTxt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hbdMsg">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>740</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: red;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgb(182, 75, 28);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>31</width>
     <height>641</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>870</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>800</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgba(107, 176, 255, 179);</string>
   </property>
   <property name="text">
    <string>PLAY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameTxt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgb(182, 75, 28);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgb(182, 75, 28);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgb(182, 75, 28);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>Welcome to the Game</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>20</width>
     <height>641</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="ageTxt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgb(182, 75, 28);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>High Score</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="highScoreTxt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>660</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgb(182, 75, 28);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>Last 3 Score</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="score1txt">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>660</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="score2txt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>660</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="score3txt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>660</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 35pt &quot;Academy Engraved LET&quot;;
background-color: rgb(182, 75, 28);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>Ranking</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="rankingTxt">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(82, 217, 255, 128);
border: 5px solid grey;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
